kvmagent/kvmagent/plugins/virtiofs_plugin.py

```
@@ -0,0 +1,425 @@
+        if conn is None:
+            raise Exception("Failed to open libvirt connection")
+
+        # Try to find domain by UUID
+        try:
+            domain = conn.lookupByUUIDString(vm_uuid)
+            return domain, conn
+        except libvirt.libvirtError:
+            # Try to find by name (zstack uses uuid as name)
+            domain = conn.lookupByName(vm_uuid)
+            return domain, conn
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)